### ISSUE TITLE
Invalid pagination token given.

### ISSUE THREAD

**Issue by dimonchik-com** [aws-amplify/amplify-cli#3459]
I have next shema 
```graphql
type Word @model
@key(name: "ByCollectionOwnerWord",
  fields: ["owner", "myCreatedAt"],
  queryField: "selectByOwnerCollectionWord")
@auth(
  rules: [
    {allow: owner, ownerField: "owner", operations: [create, update, delete]},
  ]
)
{
  id: ID!
  en: String!
  ru: String
  statusLearn: Boolean
  isAddInWorkout: Boolean
  collectionID: String!
  owner: String!
  myCreatedAt:String!
  myUpdatedAt:String!
}
```
when I try get data using **selectByOwnerCollectionWord** and limit i got next error 
com.amazonaws.deepdish.common.pagination.InvalidPaginationTokenException: Invalid pagination token given.

**Comment by ammarkarachi** [comment]
@dimonchik-com what version of Amplify CLI are you using?

**Comment by ammarkarachi** [comment]
@dimonchik-com The fix should be available 4.13.4

**Comment by santi8194** [comment]
I still have this issue when using the queryField after upgrading to 4.29.4.  @dimonchik-com Did you ever find a solution?
